src/dbt/kipptaf/models/amplify/intermediate/int_amplify__all_assessments.sql

```diff
@@ -0,0 +1,181 @@
+    ),
+
+    overall_composite_by_window as (
+        select
+            mclass_academic_year,
+            mclass_student_number,
+
+            coalesce(p.boy, 'No data') as boy,
+            coalesce(p.moy, 'No data') as moy,
+            coalesce(p.eoy, 'No data') as eoy,
+        from
+            composite_only pivot (
+                max(mclass_measure_level) for mclass_period in ('BOY', 'MOY', 'EOY')
+            ) as p
+    ),
+
+    probe_eligible_tag as (
+        select
+            s.mclass_academic_year,
+            s.mclass_student_number,
+
+            c.boy,
+            c.moy,
+            c.eoy,
+
+            if(
+                c.boy in ('Below Benchmark', 'Well Below Benchmark'), 'Yes', 'No'
+            ) as boy_probe_eligible,
+
+            if(
+                c.moy in ('Below Benchmark', 'Well Below Benchmark'), 'Yes', 'No'
+            ) as moy_probe_eligible,
+        from assessments_scores as s
+        left join
+            overall_composite_by_window as c
+            on s.mclass_academic_year = c.mclass_academic_year
+            and s.mclass_student_number = c.mclass_student_number
+    )
+
+select
+    s.mclass_academic_year,
+    s.mclass_student_number,
+    s.assessment_type,
+    s.mclass_assessment_grade,
+    s.mclass_period,
+    s.mclass_client_date,
+    s.mclass_sync_date,
+    s.mclass_measure,
+    s.mclass_measure_score,
+    s.mclass_measure_level,
+    s.mclass_measure_level_int,
+    s.mclass_measure_percentile,
+    s.mclass_measure_semester_growth,
+    s.mclass_measure_year_growth,
+    s.mclass_probe_number,
+    s.mclass_total_number_of_probes,
+    s.mclass_score_change,
+
+    p.boy_probe_eligible,
+    p.moy_probe_eligible,
+    p.boy as boy_composite,
+    p.moy as moy_composite,
+    p.eoy as eoy_composite,
+
+    case
+        when p.boy_probe_eligible = 'Yes' and s.mclass_period = 'BOY->MOY'
+        then 'Yes'
+        when p.moy_probe_eligible = 'Yes' and s.mclass_period = 'MOY->EOY'
+        then 'Yes'
+        when p.boy_probe_eligible = 'No' and s.mclass_period = 'BOY->MOY'
+        then 'No'
+        when p.moy_probe_eligible = 'No' and s.mclass_period = 'MOY->EOY'
+        then 'No'
+        else 'Not applicable'
+    end as pm_probe_eligible,
+
+    case
+        when
+            p.boy_probe_eligible = 'Yes'
+            and s.mclass_period = 'BOY->MOY'
+            and s.mclass_total_number_of_probes is not null
+        then 'Yes'
+        when
+            p.moy_probe_eligible = 'Yes'
+            and s.mclass_period = 'MOY->EOY'
+            and s.mclass_total_number_of_probes is not null
+        then 'Yes'
+        when
+            p.boy_probe_eligible = 'No'
+            and s.mclass_period = 'BOY->MOY'
+            and s.mclass_total_number_of_probes is null
+        then 'No'
+        when
+            p.moy_probe_eligible = 'No'
+            and s.mclass_period = 'MOY->EOY'
+            and s.mclass_total_number_of_probes is null
+        then 'No'

```

Comment:
@GabyRangelB these two statements were identical to the above. I changed them to `= 'No'`. Want to confirm that was the right thing to do.

Reply:
I've only had one coffee so far, so I'm a bit dense rn... how were they similar before? They were all = 'Yes' but the third condition was is not null on the first two, and is null on the second two. My intent with this case statement was to check if PM eligible kiddos have tried at least one PM. If they haven't (is null), then they get tagged as 'y'all missing a test!'. Then, anyone who is not eligible for PM testing, we just say 'Not applicable.' Is my logic to achieve that off?

Reply:
ohhhh my bad